CONTRIBUTING.md

```diff
@@ -0,0 +1,29 @@
+# Contributing to `iiif-prezi3`
+
+Issues and pull requests are appreciated. If you are working on a pull request please ensure you create an issue first so the implementation details can be discussed with other interested parties.
+
+## Submitting issues
+
+In the case of bugs please describe in detail, preferably with a link to the Manifest and/or other IIIF asset that demonstrates the problem.
+
+If you suggest a new feature, please give an example of expected behaviour and use. For changes to modules and methods a test case would be ideal.
+
+## Coding style
+
+If submitting a pull request:
+
+   * Please discuss in an issue before submitting a pull request for significant changes.
+   * Follow [PEP8](https://www.python.org/dev/peps/pep-0008/) and [PEP257](https://www.python.org/dev/peps/pep-0257/). These will be required in the CI builds.
+   * Don't repeat code.
+   * Cover the code with tests.
```

Comment:
Run the linting before committing the code by running `tox`

Reply:
Once #53 is merged, running `tox` will also autofix linting issues.